single.php

```diff
@@ -16,14 +16,6 @@
 
 			get_template_part( 'content', 'single' );
 
-			/**
-			 * Functions hooked in to storefront_single_post_after action
-			 *
-			 * @hooked storefront_post_nav         - 10
-			 * @hooked storefront_display_comments - 20
-			 */
-			do_action( 'storefront_single_post_after' );
```

Comment:
I got a little doubt and checked the article and comments structure of HTML5 Doctor - http://html5doctor.com/understanding-aside/
Which is the same as I've brought here, hopefully.